### PR TITLE
abseil-cpp: remove cxx standard flag

### PR DESCRIPTION
**Pull request by cpcloud**
###### Motivation for this change

This PR removes `-DCMAKE_CXX_STANDARD=17` from abseil-cpp while preserving the ability
to set it with `.override`.

Adding this flag to `abseil-cpp` forces all dependents to also set this flag to `17`,
which isn't always possible.

`or-tools` seems to be the only package that _requires_ `-DCMAKE_CXX_STANDARD=17`, because
it builds using `-std=c++17`.

I've got a [package upgrade PR](https://github.com/NixOS/nixpkgs/pull/143693)
that will not build with `-DCMAKE_CXX_STANDARD=17`, and one more in progress that depends on #143693
(both also depend on gRPC as well).

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/grpc/default.nix
Comment (SuperSandro2000) [inline pkgs/development/libraries/grpc/default.nix:63]:
```suggestion
  ] ++ lib.optionals ((stdenv.hostPlatform.useLLVM or false) && lib.versionOlder stdenv.cc.cc.version "11.0") [
```

File: pkgs/development/libraries/abseil-cpp/default.nix
Comment (SuperSandro2000) [inline pkgs/development/libraries/abseil-cpp/default.nix:7]:
```suggestion
, cxxStandard ? "17"
```

File: pkgs/development/libraries/abseil-cpp/default.nix
Comment (SuperSandro2000) [inline pkgs/development/libraries/abseil-cpp/default.nix:32]:
```suggestion
```

File: pkgs/top-level/all-packages.nix
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:31173]:
```suggestion
      cxxStandard = "17";
```

File: pkgs/development/libraries/abseil-cpp/default.nix
Comment (SuperSandro2000) [inline pkgs/development/libraries/abseil-cpp/default.nix:33]:
```suggestion
    "-DCMAKE_CXX_STANDARD=${cxxStandard}"
```

**Comment by risicle** [comment]
`grpc` builds successfully, macos 10.15.

File: pkgs/development/libraries/abseil-cpp/default.nix
Comment (cpcloud) [reply pkgs/development/libraries/abseil-cpp/default.nix:7]: I don't think this should be the default. One of the purposes of this change is to give `abseil-cpp` the default behavior of `-DCMAKE_CXX_STANDARD`, and not force transitive overrides unless necessary, which is so far only in the case of `or-tools`.

**Comment by veprbl** [comment]
This could use some commit squashing

**Comment by cpcloud** [comment]
@veprbl Yup, I was waiting for approval.

**Comment by cpcloud** [comment]
@SuperSandro2000 @veprbl Squashed a couple commits. This is ready for another rough of review.

**Review by veprbl — APPROVED** [review]
Adding a whole new option is a bit of an overkill – an `overrideAttrs` would be enough, IMO, but this is a very reasonable change, nevertheless.

**Comment by veprbl** [comment]
Result of `nixpkgs-review pr 143967` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>23 packages marked as broken and skipped:</summary>
  <ul>
    <li>ledger2beancount</li>
    <li>python38Packages.arviz</li>
    <li>python38Packages.einops</li>
    <li>python38Packages.graph_nets</li>
    <li>python38Packages.mask-rcnn</li>
    <li>python38Packages.oauthenticator</li>
    <li>python38Packages.optuna</li>
    <li>python38Packages.pymc3</li>
    <li>python38Packages.rl-coach</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.tensorflow-probability</li>
    <li>python39Packages.arviz</li>
    <li>python39Packages.einops</li>
    <li>python39Packages.graph_nets</li>
    <li>python39Packages.mask-rcnn</li>
    <li>python39Packages.oauthenticator</li>
    <li>python39Packages.optuna</li>
    <li>python39Packages.pymc3</li>
    <li>python39Packages.rl-coach</li>
    <li>python39Packages.spacy-transformers</li>
    <li>python39Packages.tensorflow-probability</li>
    <li>sysdig</li>
    <li>tts</li>
  </ul>
</details>
<details>
  <summary>52 packages failed to build:</summary>
  <ul>
    <li>beancount (python39Packages.beancount)</li>
    <li>duplicity</li>
    <li>duply</li>
    <li>dvc-with-remotes</li>
    <li>fava</li>
    <li>libtensorflow</li>
    <li>lieer</li>
    <li>python38Packages.baselines</li>
    <li>python38Packages.beancount</li>
    <li>python38Packages.beancount_docverif</li>
    <li>python38Packages.cirq</li>
    <li>python38Packages.cirq-google</li>
    <li>python38Packages.edward</li>
    <li>python38Packages.gcsfs</li>
    <li>python38Packages.gensim</li>
    <li>python38Packages.google-cloud-automl</li>
    <li>python38Packages.google-cloud-bigquery</li>
    <li>python38Packages.google-cloud-error-reporting</li>
    <li>python38Packages.google-cloud-logging</li>
    <li>python38Packages.google-cloud-monitoring</li>
    <li>python38Packages.google-cloud-storage</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.scikit-tda</li>
    <li>python38Packages.smart-open</li>
    <li>python38Packages.spacy</li>
    <li>python38Packages.tensorflow (python38Packages.tensorflow-build ,python38Packages.tensorflow-build_2 ,python38Packages.tensorflowWithoutCuda ,python38Packages.tensorflow_2)</li>
    <li>python38Packages.textacy</li>
    <li>python38Packages.tflearn</li>
    <li>python38Packages.umap-learn</li>
    <li>python39Packages.baselines</li>
    <li>python39Packages.beancount_docverif</li>
    <li>python39Packages.cirq</li>
    <li>python39Packages.cirq-google</li>
    <li>python39Packages.edward</li>
    <li>python39Packages.gcsfs</li>
    <li>python39Packages.gensim</li>
    <li>python39Packages.google-cloud-automl</li>
    <li>python39Packages.google-cloud-bigquery</li>
    <li>python39Packages.google-cloud-error-reporting</li>
    <li>python39Packages.google-cloud-logging</li>
    <li>python39Packages.google-cloud-monitoring</li>
    <li>python39Packages.google-cloud-storage</li>
    <li>python39Packages.pathy</li>
    <li>python39Packages.scikit-tda</li>
    <li>python39Packages.smart-open</li>
    <li>python39Packages.spacy</li>
    <li>python39Packages.tensorflow (python39Packages.tensorflow-build ,python39Packages.tensorflow-build_2 ,python39Packages.tensorflowWithoutCuda ,python39Packages.tensorflow_2)</li>
    <li>python39Packages.textacy</li>
    <li>python39Packages.tflearn</li>
    <li>python39Packages.umap-learn</li>
    <li>qv2ray</li>
    <li>snakemake</li>
  </ul>
</details>
<details>
  <summary>105 packages built:</summary>
  <ul>
    <li>abseil-cpp</li>
    <li>bear</li>
    <li>charge-lnd</li>
    <li>gcalcli</li>
    <li>git-annex-remote-googledrive</li>
    <li>grpc</li>
    <li>lndmanage</li>
    <li>python38Packages.certbot-dns-google</li>
    <li>python38Packages.chirpstack-api</li>
    <li>python38Packages.drivelib</li>
    <li>python38Packages.goobook</li>
    <li>python38Packages.google-api-core</li>
    <li>python38Packages.google-api-python-client</li>
    <li>python38Packages.google-cloud-access-context-manager</li>
    <li>python38Packages.google-cloud-appengine-logging</li>
    <li>python38Packages.google-cloud-asset</li>
    <li>python38Packages.google-cloud-audit-log</li>
    <li>python38Packages.google-cloud-bigquery-datatransfer</li>
    <li>python38Packages.google-cloud-bigquery-logging</li>
    <li>python38Packages.google-cloud-bigtable</li>
    <li>python38Packages.google-cloud-container</li>
    <li>python38Packages.google-cloud-core</li>
    <li>python38Packages.google-cloud-datacatalog</li>
    <li>python38Packages.google-cloud-dataproc</li>
    <li>python38Packages.google-cloud-datastore</li>
    <li>python38Packages.google-cloud-dlp</li>
    <li>python38Packages.google-cloud-dns</li>
    <li>python38Packages.google-cloud-firestore</li>
    <li>python38Packages.google-cloud-iam</li>
    <li>python38Packages.google-cloud-iam-logging</li>
    <li>python38Packages.google-cloud-iot</li>
    <li>python38Packages.google-cloud-kms</li>
    <li>python38Packages.google-cloud-language</li>
    <li>python38Packages.google-cloud-org-policy</li>
    <li>python38Packages.google-cloud-os-config</li>
    <li>python38Packages.google-cloud-pubsub</li>
    <li>python38Packages.google-cloud-redis</li>
    <li>python38Packages.google-cloud-resource-manager</li>
    <li>python38Packages.google-cloud-runtimeconfig</li>
    <li>python38Packages.google-cloud-secret-manager</li>
    <li>python38Packages.google-cloud-securitycenter</li>
    <li>python38Packages.google-cloud-spanner</li>
    <li>python38Packages.google-cloud-speech</li>
    <li>python38Packages.google-cloud-tasks</li>
    <li>python38Packages.google-cloud-texttospeech</li>
    <li>python38Packages.google-cloud-trace</li>
    <li>python38Packages.google-cloud-translate</li>
    <li>python38Packages.google-cloud-videointelligence</li>
    <li>python38Packages.google-cloud-vision</li>
    <li>python38Packages.google-cloud-websecurityscanner</li>
    <li>python38Packages.googleapis-common-protos</li>
    <li>python38Packages.grpc-google-iam-v1</li>
    <li>python38Packages.proto-plus</li>
    <li>python38Packages.pydrive</li>
    <li>python38Packages.weboob</li>
    <li>python38Packages.woob</li>
    <li>python39Packages.certbot-dns-google</li>
    <li>python39Packages.chirpstack-api</li>
    <li>python39Packages.drivelib</li>
    <li>python39Packages.goobook</li>
    <li>python39Packages.google-api-core</li>
    <li>python39Packages.google-api-python-client</li>
    <li>python39Packages.google-cloud-access-context-manager</li>
    <li>python39Packages.google-cloud-appengine-logging</li>
    <li>python39Packages.google-cloud-asset</li>
    <li>python39Packages.google-cloud-audit-log</li>
    <li>python39Packages.google-cloud-bigquery-datatransfer</li>
    <li>python39Packages.google-cloud-bigquery-logging</li>
    <li>python39Packages.google-cloud-bigtable</li>
    <li>python39Packages.google-cloud-container</li>
    <li>python39Packages.google-cloud-core</li>
    <li>python39Packages.google-cloud-datacatalog</li>
    <li>python39Packages.google-cloud-dataproc</li>
    <li>python39Packages.google-cloud-datastore</li>
    <li>python39Packages.google-cloud-dlp</li>
    <li>python39Packages.google-cloud-dns</li>
    <li>python39Packages.google-cloud-firestore</li>
    <li>python39Packages.google-cloud-iam</li>
    <li>python39Packages.google-cloud-iam-logging</li>
    <li>python39Packages.google-cloud-iot</li>
    <li>python39Packages.google-cloud-kms</li>
    <li>python39Packages.google-cloud-language</li>
    <li>python39Packages.google-cloud-org-policy</li>
    <li>python39Packages.google-cloud-os-config</li>
    <li>python39Packages.google-cloud-pubsub</li>
    <li>python39Packages.google-cloud-redis</li>
    <li>python39Packages.google-cloud-resource-manager</li>
    <li>python39Packages.google-cloud-runtimeconfig</li>
    <li>python39Packages.google-cloud-secret-manager</li>
    <li>python39Packages.google-cloud-securitycenter</li>
    <li>python39Packages.google-cloud-spanner</li>
    <li>python39Packages.google-cloud-speech</li>
    <li>python39Packages.google-cloud-tasks</li>
    <li>python39Packages.google-cloud-texttospeech</li>
    <li>python39Packages.google-cloud-trace</li>
    <li>python39Packages.google-cloud-translate</li>
    <li>python39Packages.google-cloud-videointelligence</li>
    <li>python39Packages.google-cloud-vision</li>
    <li>python39Packages.google-cloud-websecurityscanner</li>
    <li>python39Packages.googleapis-common-protos</li>
    <li>python39Packages.grpc-google-iam-v1</li>
    <li>python39Packages.proto-plus</li>
    <li>python39Packages.pydrive</li>
    <li>python39Packages.weboob</li>
    <li>python39Packages.woob</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 143967` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
18 packages marked as broken and skipped:
</summary>
<ul>
<li>linuxKernel.packages.hardkernel_4_14.sysdig</li>
<li>linuxKernel.packages.linux_4_4.sysdig</li>
<li>linuxKernel.packages.linux_4_9.sysdig</li>
<li>linuxKernel.packages.linux_testing_bcachefs.sysdig</li>
<li>linuxPackages_hardkernel_latest.sysdig</li>
<li>linuxPackages_testing_bcachefs.sysdig</li>
<li>python38Packages.arviz</li>
<li>python38Packages.dm-sonnet</li>
<li>python38Packages.graph_nets</li>
<li>python38Packages.optuna</li>
<li>python38Packages.pymc3</li>
<li>python38Packages.rl-coach</li>
<li>python39Packages.arviz</li>
<li>python39Packages.dm-sonnet</li>
<li>python39Packages.graph_nets</li>
<li>python39Packages.optuna</li>
<li>python39Packages.pymc3</li>
<li>python39Packages.rl-coach</li>
</ul>
</details>
<details>

  <summary>9 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>linuxPackages_hardened.sysdig: log was empty</li>
<li>linuxPackages_latest-libre.sysdig: log was empty</li>
<li>python38Packages.einops: log was empty</li>
<li>python38Packages.google-cloud-error-reporting: log was empty</li>
<li>python38Packages.google-cloud-logging: log was empty</li>
<li>python39Packages.einops: log was empty</li>
<li>python39Packages.google-cloud-error-reporting: log was empty</li>
<li>python39Packages.google-cloud-logging: log was empty</li>
<li>rippled-validator-keys-tool: log was empty</li>
  </ul>
</details>
<details>

  <summary>4 packages failed to build and are new build failures:</summary>
  <ul>
<li>fcitx5-mozc: log was empty</li>
<li>(linuxPackages_5_10_hardened.sysdig): log was empty</li>
<li>(linuxKernel.packages.linux_latest_libre.sysdig): log was empty</li>
<li>tdesktop: log was empty</li>
  </ul>
</details>
<details>
<summary>
169 packages built:
</summary>
<ul>
<li>abseil-cpp</li>
<li>beancount (python39Packages.beancount)</li>
<li>bear</li>
<li>charge-lnd</li>
<li>deja-dup</li>
<li>duplicity</li>
<li>duply</li>
<li>dvc-with-remotes</li>
<li>fava</li>
<li>gcalcli</li>
<li>git-annex-remote-googledrive</li>
<li>google-cloud-cpp</li>
<li>grpc</li>
<li>home-assistant</li>
<li>kmymoney</li>
<li>ledger2beancount</li>
<li>lieer</li>
<li>linuxKernel.packages.linux_4_14.sysdig</li>
<li>linuxPackages_4_14_hardened.sysdig (linuxKernel.packages.linux_4_14_hardened.sysdig)</li>
<li>linuxKernel.packages.linux_4_19.sysdig</li>
<li>linuxPackages_4_19_hardened.sysdig (linuxKernel.packages.linux_4_19_hardened.sysdig)</li>
<li>linuxPackages.sysdig (linuxKernel.packages.linux_5_10.sysdig)</li>
<li>linuxKernel.packages.linux_5_14.sysdig</li>
<li>linuxPackages_5_14_hardened.sysdig (linuxKernel.packages.linux_5_14_hardened.sysdig)</li>
<li>linuxPackages_latest.sysdig (linuxKernel.packages.linux_5_15.sysdig)</li>
<li>linuxKernel.packages.linux_5_4.sysdig</li>
<li>linuxPackages_5_4_hardened.sysdig (linuxKernel.packages.linux_5_4_hardened.sysdig)</li>
<li>linuxPackages-libre.sysdig (linuxKernel.packages.linux_libre.sysdig)</li>
<li>linuxPackages_lqx.sysdig (linuxKernel.packages.linux_lqx.sysdig)</li>
<li>linuxPackages_xanmod.sysdig (linuxKernel.packages.linux_xanmod.sysdig)</li>
<li>linuxPackages_zen.sysdig (linuxKernel.packages.linux_zen.sysdig)</li>
<li>lndmanage</li>
<li>murmur</li>
<li>or-tools</li>
<li>python38Packages.beancount</li>
<li>python38Packages.beancount_docverif</li>
<li>python38Packages.certbot-dns-google</li>
<li>python38Packages.chirpstack-api</li>
<li>python38Packages.cirq</li>
<li>python38Packages.cirq-google</li>
<li>python38Packages.drivelib</li>
<li>python38Packages.gcsfs</li>
<li>python38Packages.gensim</li>
<li>python38Packages.goobook</li>
<li>python38Packages.google-api-core</li>
<li>python38Packages.google-api-python-client</li>
<li>python38Packages.google-cloud-access-context-manager</li>
<li>python38Packages.google-cloud-appengine-logging</li>
<li>python38Packages.google-cloud-asset</li>
<li>python38Packages.google-cloud-audit-log</li>
<li>python38Packages.google-cloud-automl</li>
<li>python38Packages.google-cloud-bigquery</li>
<li>python38Packages.google-cloud-bigquery-datatransfer</li>
<li>python38Packages.google-cloud-bigquery-logging</li>
<li>python38Packages.google-cloud-bigtable</li>
<li>python38Packages.google-cloud-container</li>
<li>python38Packages.google-cloud-core</li>
<li>python38Packages.google-cloud-datacatalog</li>
<li>python38Packages.google-cloud-dataproc</li>
<li>python38Packages.google-cloud-datastore</li>
<li>python38Packages.google-cloud-dlp</li>
<li>python38Packages.google-cloud-dns</li>
<li>python38Packages.google-cloud-firestore</li>
<li>python38Packages.google-cloud-iam</li>
<li>python38Packages.google-cloud-iam-logging</li>
<li>python38Packages.google-cloud-iot</li>
<li>python38Packages.google-cloud-kms</li>
<li>python38Packages.google-cloud-language</li>
<li>python38Packages.google-cloud-monitoring</li>
<li>python38Packages.google-cloud-org-policy</li>
<li>python38Packages.google-cloud-os-config</li>
<li>python38Packages.google-cloud-pubsub</li>
<li>python38Packages.google-cloud-redis</li>
<li>python38Packages.google-cloud-resource-manager</li>
<li>python38Packages.google-cloud-runtimeconfig</li>
<li>python38Packages.google-cloud-secret-manager</li>
<li>python38Packages.google-cloud-securitycenter</li>
<li>python38Packages.google-cloud-spanner</li>
<li>python38Packages.google-cloud-speech</li>
<li>python38Packages.google-cloud-storage</li>
<li>python38Packages.google-cloud-tasks</li>
<li>python38Packages.google-cloud-texttospeech</li>
<li>python38Packages.google-cloud-trace</li>
<li>python38Packages.google-cloud-translate</li>
<li>python38Packages.google-cloud-videointelligence</li>
<li>python38Packages.google-cloud-vision</li>
<li>python38Packages.google-cloud-websecurityscanner</li>
<li>python38Packages.googleapis-common-protos</li>
<li>python38Packages.grpc-google-iam-v1</li>
<li>python38Packages.oauthenticator</li>
<li>python38Packages.ortools</li>
<li>python38Packages.pathy</li>
<li>python38Packages.proto-plus</li>
<li>python38Packages.pydrive</li>
<li>python38Packages.smart-open</li>
<li>python38Packages.spacy</li>
<li>python38Packages.spacy-transformers</li>
<li>python38Packages.textacy</li>
<li>python38Packages.weboob</li>
<li>python38Packages.woob</li>
<li>python39Packages.beancount_docverif</li>
<li>python39Packages.certbot-dns-google</li>
<li>python39Packages.chirpstack-api</li>
<li>python39Packages.cirq</li>
<li>python39Packages.cirq-google</li>
<li>python39Packages.drivelib</li>
<li>python39Packages.gcsfs</li>
<li>python39Packages.gensim</li>
<li>python39Packages.goobook</li>
<li>python39Packages.google-api-core</li>
<li>python39Packages.google-api-python-client</li>
<li>python39Packages.google-cloud-access-context-manager</li>
<li>python39Packages.google-cloud-appengine-logging</li>
<li>python39Packages.google-cloud-asset</li>
<li>python39Packages.google-cloud-audit-log</li>
<li>python39Packages.google-cloud-automl</li>
<li>python39Packages.google-cloud-bigquery</li>
<li>python39Packages.google-cloud-bigquery-datatransfer</li>
<li>python39Packages.google-cloud-bigquery-logging</li>
<li>python39Packages.google-cloud-bigtable</li>
<li>python39Packages.google-cloud-container</li>
<li>python39Packages.google-cloud-core</li>
<li>python39Packages.google-cloud-datacatalog</li>
<li>python39Packages.google-cloud-dataproc</li>
<li>python39Packages.google-cloud-datastore</li>
<li>python39Packages.google-cloud-dlp</li>
<li>python39Packages.google-cloud-dns</li>
<li>python39Packages.google-cloud-firestore</li>
<li>python39Packages.google-cloud-iam</li>
<li>python39Packages.google-cloud-iam-logging</li>
<li>python39Packages.google-cloud-iot</li>
<li>python39Packages.google-cloud-kms</li>
<li>python39Packages.google-cloud-language</li>
<li>python39Packages.google-cloud-monitoring</li>
<li>python39Packages.google-cloud-org-policy</li>
<li>python39Packages.google-cloud-os-config</li>
<li>python39Packages.google-cloud-pubsub</li>
<li>python39Packages.google-cloud-redis</li>
<li>python39Packages.google-cloud-resource-manager</li>
<li>python39Packages.google-cloud-runtimeconfig</li>
<li>python39Packages.google-cloud-secret-manager</li>
<li>python39Packages.google-cloud-securitycenter</li>
<li>python39Packages.google-cloud-spanner</li>
<li>python39Packages.google-cloud-speech</li>
<li>python39Packages.google-cloud-storage</li>
<li>python39Packages.google-cloud-tasks</li>
<li>python39Packages.google-cloud-texttospeech</li>
<li>python39Packages.google-cloud-trace</li>
<li>python39Packages.google-cloud-translate</li>
<li>python39Packages.google-cloud-videointelligence</li>
<li>python39Packages.google-cloud-vision</li>
<li>python39Packages.google-cloud-websecurityscanner</li>
<li>python39Packages.googleapis-common-protos</li>
<li>python39Packages.grpc-google-iam-v1</li>
<li>python39Packages.oauthenticator</li>
<li>python39Packages.ortools</li>
<li>python39Packages.pathy</li>
<li>python39Packages.proto-plus</li>
<li>python39Packages.pydrive</li>
<li>python39Packages.smart-open</li>
<li>python39Packages.spacy</li>
<li>python39Packages.spacy-transformers</li>
<li>python39Packages.textacy</li>
<li>python39Packages.weboob</li>
<li>python39Packages.woob</li>
<li>qv2ray</li>
<li>rippled</li>
<li>snakemake</li>
<li>sysdig</li>
</ul>
</details>

**Comment by cpcloud** [comment]
@veprbl @SuperSandro2000 I'm not totally following what's breaking here. It looks like a few google-cloud packages are breaking, but they all seem to be broken on master too.

How can I tell what's newly breaking in this PR vs master?

**Comment by SuperSandro2000** [comment]
I am already taking care of this.

**Comment by SuperSandro2000** [comment]
So, this can be merged when ofborg is done.

**Comment by cpcloud** [comment]
@SuperSandro2000 Thanks for fixing! Much appreciated.

**Comment by cpcloud** [comment]
@SuperSandro2000 I guess this is done? I'm assuming the darwin builds are hanging due to this https://github.com/NixOS/nixpkgs/issues/137678?

**Comment by SuperSandro2000** [comment]
I don't know why more Darwin packages fail to be build exactly but there seems to be a lot of unrelated issues there which we are not going to investigate all in this pr.

**Comment by cpcloud** [comment]
@SuperSandro2000 Thank you.

**Pull request merged**
